spec/services/tile_metadata_service_spec.rb

```diff
@@ -63,14 +65,6 @@
         expect(path).to eq(cloud_path.join("33", "1d", "70", "331d70a54bd94a6580e4763c8f6b34fd", "mosaic-#{fingerprint}.json").to_s)
       end
     end
-
-    context "when there aren't any files on the raster members" do
```

Comment:
Why don't we want a test for this case anymore?

Reply:
Good catch. We do, I just needed to move the `raise Error` elsewhere.